Configuration/PyReleaseValidation/python/relval_upgrade.py

```diff
@@ -31,8 +31,11 @@ def makeStepName(key,frag,step,suffix):
             for step in upgradeProperties[year][key]['ScenToRun']:                    
                 stepMaker = makeStepName
                 if 'Sim' in step:
-                    if 'HLBeamSpot' in step and '14TeV' in frag:
-                        step = 'GenSimHLBeamSpot14'
+                    if 'HLBeamSpot' in step:
+                        if '14TeV' in frag:
+                            step = 'GenSimHLBeamSpot14'
+                        if 'CloseByParticle' in frag or 'CE_E' in frag or 'CE_H' in frag:
+                            step = 'GenSimHLBeamSpot14HGCALCloseBy'
```

Comment:
Given the name of the step, shouldn't this be made conditional to `if '14TeV' in frag:`?

Reply:
I think the CloseBy "beats" the 14TeV in terms of importance and it is also not restricted to 14TeV. Maybe the step could be called "GenSimHGCALCloseBy" rather than "GenSimHLBeamSpot14HGCALCloseBy" because it's the exact point of the HGCALCloseBy to turn off anything HLBeamSpot-like (and use a module without any smearing instead), but I think it's fine - for my taste.  When inventing names and matching them between configs, you'll necessarily end up with some slight descriptive imprecisions.

Reply:
> When inventing names and matching them between configs, you'll necessarily end up with some slight descriptive imprecisions.

I agree. But we are still in time to fix it here,
If it is not restricted to 14 TeV, please remove it from the name.

Reply:
By the way: could you please remind me what "CE_E" and "CE_H" are, and why vertex smearing should be disabled also for them? A line in the PR description with it can be useful.

Reply:
Thanks, I removed the 14TeV. CE_E and CE_H are the two sections of HGCAL. There are some workflows that use the CloseByParticleGun as well:

https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1387-L1394